mocks/org_config.go

```diff
@@ -0,0 +1,22 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/influxdata/chronograf"
+)
+
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+type OrganizationConfigStore struct {
+	FindOrCreateF func(ctx context.Context, id string) (*chronograf.OrganizationConfig, error)
+	UpdateF       func(ctx context.Context, target *chronograf.OrganizationConfig) error
+}
+
+func (oc *OrganizationConfigStore) FindOrCreate(ctx context.Context, id string) (*chronograf.OrganizationConfig, error) {
```

Comment:
*super nit* prefer single character method receiver names that reflect the noun part of the type of the method receiver. e.g.

```
w -> ResponseWriter
s  -> OrganizationConfigStore
s -> BucketService
```

Reply:
makes sense.. but with exceptions for `ctx` and `id`?